modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -1,4 +1,4 @@
-= Configure OpenID Connect Client Management
+= Configuring OpenID Connect Client Management
 ifndef::env-site,env-github[]
```

Comment:
FWIW, I think you can remove all these attribute lines...

Reply:
Yeah, we can. I'll do that since I'm there anyway.